CHANGELOG.md

```diff
@@ -54,6 +54,8 @@ Current
 - [Fix metric and dimension names for wikipedia-example](https://github.com/yahoo/fili/pull/415)
     * The metrics and dimensions configured in the `fili-wikipedia-example` were different from those in Druid and as a result the queries sent to Druid were incorrect
 
+- [Log stack trace at error on unexpected DimensionServlet failures]()
```

Comment:
Maybe document 

> Latest changes all appear to be at the top of list

 in the [CHANGELOG process](https://github.com/yahoo/fili/pull/426)?